andela/blackninjas-frontend
# #169817824 Implementing trip request comments

#### What does this PR do?
- Users should be able to comment on Travel requests (UI)

#### Description of Task to be completed?
- User is able to post comment to a travel request to give additional information
- User is able to see the posted comment within the thread

#### How should this be manually tested?
After clone repository, go to the root of the project open path into command run the `npm install` and once setup is done click `npm start-dev`, this will start a local server which will then automatically open your default browser, then navigate to `http://localhost:8080/trips` and the click on a given trip to see full details.

#### What are the relevant pivotal tracker stories?
- [#169817824](https://www.pivotaltracker.com/story/show/169817824)

#### Screenshots (if appropriate)

<img width="1440" alt="Screen Shot 2020-02-28 at 14 00 42" src="https://user-images.githubusercontent.com/28502544/75547121-e7150300-5a32-11ea-9f30-04223fe3ca59.png">
<img width="1440" alt="Screen Shot 2020-02-28 at 14 00 50" src="https://user-images.githubusercontent.com/28502544/75547131-ec724d80-5a32-11ea-87a2-b3b3914a6f78.png">
<img width="405" alt="Screen Shot 2020-02-28 at 14 01 24" src="https://user-images.githubusercontent.com/28502544/75547133-ed0ae400-5a32-11ea-863a-6afad9f20220.png">
<img width="401" alt="Screen Shot 2020-02-28 at 14 01 42" src="https://user-images.githubusercontent.com/28502544/75547136-eda37a80-5a32-11ea-8f40-edd9b6da01f0.png">
